### DISCUSSION TITLE
Daily VIIRS Black Marble - VNP46A2

### DISCUSSION THREAD

**Discussion by RaananHadar** [microsoft/PlanetaryComputer#88] [Data request]
[Black Marble](https://viirsland.gsfc.nasa.gov/Products/NASA/BlackMarble.html) is a mature product used by NASA that was relatively recently publicly released in the last year. Even though this is a new public product, the archive goes all back to 2012, showcasing the internal value of Black Marble to NASA.

Being the best available public source of daily night light data and benefiting from special pre-processing algorithms ([AKA "Black Marble"](https://viirsland.gsfc.nasa.gov/PDF/VIIRS_BlackMarbleUserGuide_V1.1.pdf)), it boasts much a much higher radiometric quality and atmospheric clarity and value from the standard VIIRS night lights product. 

It is very useful for studying of patterns of life and maritime activity. I also believe that increasing the temporal resolution to daily and getting more recent collection than the current HREA data which goes only until 2019 has many advantages and benefits that for studying the planet.

Lastly, I think that Planetary Computer has a special benefit here to bring to the table: modernization. To date, the dataset is available mostly as h5 files and access is somewhat quirky which can scare off many 'science' level professionals without sufficient level of data engineering skills. It is also worth noting that black marble is curretly not available in Google Earth Engine. Ingesting black marble in a modern way in planetary computer, in a COG format and giving it planetary computer compute scale is a world's first for the dataset. This opens up many research opportunities and can really increase planetary computer's adoption.

In summary, I believe that adding black marble to the Planetary Computer catalogue will be a win-win situation for the scientific community and Microsoft.

Please do consider 🙏

**Comment by TomAugspurger** [comment]
Thanks for opening this request.

Black Marble, along with some other collections from VIIRS, is on our list (see https://github.com/stactools-packages/viirs, in particular https://github.com/stactools-packages/viirs/blob/4d01918b471d49aa450e8ca78e28bb306c3ea73c/src/stactools/viirs/fragments/VNP46A2/collection.json, for work on the stactools side).
